lexers/l/testdata/limbo_basic.b

```diff
@@ -0,0 +1,35 @@
+implement Values;
+
+include "sys.m";
+include "draw.m";
+
+sys: Sys;
+print, sprint: import sys;
+
+Values: module {
+	init: fn(nil: ref Draw->Context, nil: list of string);
+};
+
+init(nil: ref Draw->Context, nil: list of string) {
+	sys = load Sys Sys->PATH;
+
+	n := 7;
+	b := big 8;
+	f := real 3.2;
```

Comment:
I guess we can shrink this file a little bit. Let's keep it focused and reduce it to the minimum